.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      max-parallel: 3
+      matrix:
+        platform: [ubuntu-latest]
+        python-version: [3.5, 3.7, 3.8, 3.9]
```

Comment:
Just want to know why py3.6 is excluded?

Reply:
@matsui528 I don't intend to enumerate all versions, but just added 3.5 for the lowest version instead